### ISSUE TITLE
Add accessibility checks into CI for the documentation site

### ISSUE THREAD

**Issue by timwright12** [department-of-veterans-affairs/va-mobile-app#8217]
Currently, we have CI processes that will run when the documentation site is updated. In looking at the deployed site, there seem to be some level of basic accessibility issues that we should be able to scan for. In this ticket we should create a point of automation that runs in a pull request, to check for accessibility issues.

This workflow run on PRs that touch the documentation site as well as merges into develop. If, on first run, there are failing accessibility issues reach out to @timwright12 to get them fixed or prioritized.

**Definition of Done**

- [ ] An accessibility check is added into the CI process for documentation PRs
- [ ] If a failure happens while running in develop, alert slack in #va-mobile-build-alerts
- [ ] Checks run over all pages in the doc site
- [ ] Evidence of the workflow passing and failing (to confirm that it works)

**Important Links**

- [Workflows directory](https://github.com/department-of-veterans-affairs/va-mobile-app/tree/develop/.github/workflows)
- [Axe-core documentation](https://github.com/dequelabs/axe-core)
- [Axe-core in github actions example](https://bolonio.medium.com/automating-the-accessibility-tests-of-your-source-code-with-github-actions-63590cdc6860)

**Comment by timwright12** [comment]
@IsraelleHub Left some comments on the PR for you

**Comment by timwright12** [comment]
@IsraelleHub the PR for this is all approved, so you can merge it, make sure it runs once, and close this ticket out

**Comment by TKDickson** [comment]
@bischoffa I assume @DJUltraTom will be assigned to QA any QART tickets, as the floater?

**Comment by TKDickson** [comment]
(not that this one needs testing. Just - something to think about for future tickets)
